packages/datasource/slack/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { GetMeshSourceOptions, YamlConfig } from '@graphql-mesh/types';
+const OpenAPIHandler = require('@graphql-mesh/openapi');
+
+const OPENAPI_SPECIFICATION: string = "https://api.slack.com/specs/openapi/v2/slack_web.json";
+
+export default class SlackHandler extends OpenAPIHandler {
+  constructor({ name, config, cache }: GetMeshSourceOptions<YamlConfig.OpenapiHandler>) {
+    config.source = OPENAPI_SPECIFICATION;
+
+    // @ts-ignore
+    config.schemaHeaders = { authorization: `Bearer ${config.token}`};
```

Comment:
Missing space after ...token} and before `. Probably Prettier is not enabled?

Reply:
`+`